Scripts/update_ncbi_codon_table.py

```diff
@@ -16,29 +19,46 @@
 import re
 
 INDENT = len("register_ncbi_table(")
+INDENT2 = INDENT + len("table={")
+
 
+def line_wrap(text, indent=0, max_len=78, string=False):
+    """Return a wrapped line if length is larger max_len.
 
-def line_wrap(text, indent=0, max_len=78):
+    The new parameter 'string' allows to wrap quoted text which is delimited
+    by single quotes. It adds " '" to the end of the line and "'" to the start
+    of the next line.
+    """
+    split_len = max_len if not string else max_len - 2
     if len(text) <= max_len:
         return text
-    line = text[:max_len]
+    line = text[:split_len]
     assert " " in line, line
     line, rest = line.rsplit(" ", 1)
-    rest = " " * indent + rest + text[max_len:]
+    # New:
+    if string:
+        line += " '"
+        rest = "'" + rest
+    rest = " " * indent + rest + text[split_len:]
     assert len(line) < max_len
     if indent + len(rest) <= max_len:
         return line + "\n" + rest
     else:
-        return line + "\n" + line_wrap(rest, max_len)
+        return line + "\n" + line_wrap(rest, indent, max_len, string)
+
 
+print("""
+##########################################################################
+# Start of auto-generated output from Scripts/update_ncbi_codon_table.py #
+##########################################################################
 
-print("##########################################################################")
-print("# Start of auto-generated output from Scripts/update_ncbi_codon_table.py #")
-print("##########################################################################")
-print("")
-print("")
+"""[1:])
 
+version = ''
 for line in open("gc.prt").readlines():
+    if not version and line.startswith("--  Version"):
+        version = line.split('Version')[1].strip()
```

Comment:
How likely is it that `Version` occurs more than once? I've found it is often better to be explicit with the number of expected `split()`s. In this case, if we only expect it once, I think simply doing `.split('Version', 1)` makes the code more robust.

Reply:
OK